ground-core/scripts/cassandra/cassandra.sql

```diff
@@ -1,134 +1,136 @@
 /* VERSIONS */
 
-create table Versions (id varchar primary key);
+create table version (
+    id varchar PRIMARY KEY
+);
 
-create table VersionSuccessors (
-    successor_id varchar primary key,
-    vfrom varchar, 
-    vto varchar
+create table version_successor (
+    id varchar PRIMARY KEY,
+    from_version_id varchar,
+    to_version_id varchar
 );
 
-create table Items (
-    id varchar primary key
+create table item (
+    id varchar PRIMARY KEY
 );
 
-create table VersionHistoryDAGs (
+create table version_history_dag (
     item_id varchar,
-    successor_id varchar,
-    primary key(item_id, successor_id)
+    version_successor_id varchar,
+    PRIMARY KEY(item_id, version_successor_id)
 );
 
 /* MODELS */
 
-create table Structures (
+create table structure (
     item_id varchar,
     name varchar,
-    primary key(item_id, name)
+    PRIMARY KEY (item_id, name)
 );
 
-create table StructureVersions (
+create table structure_version (
     id varchar primary key,
     structure_id varchar
 );
 
-create table StructureVersionItems (
-    svid varchar,
+create table structure_version_attribute (
+    structure_version_id varchar,
     key varchar,
     type varchar,
-    primary key(svid, key)
+    PRIMARY KEY (structure_version_id, key)
 );
 
-create table RichVersions (
-    id varchar primary key,
-    structure_id varchar,
+create table rich_version (
+    id varchar PRIMARY KEY,
+    structure_version_id varchar,
     reference varchar
 );
 
-create table RichVersionExternalParameters (
-    richversion_id varchar,
+create table rich_version_external_parameter (
+    rich_version_id varchar,
     key varchar,
     value varchar,
-    primary key (richversion_id, key)
+    PRIMARY KEY (rich_version_id, key)
 );
 
-create table Tags (
-    richversion_id varchar,
+create table tag (
+    rich_version_id varchar,
     key varchar,
     value varchar,
     type varchar,
-    primary key(richversion_id, key)
+    PRIMARY KEY (rich_version_id, key)
 );
 
-create table Edges (
+create table edge (
     item_id varchar,
     name varchar,
-    primary key(item_id, name)
+    PRIMARY KEY (item_id, name)
 );
 
-create table Nodes (
+create table node (
     item_id varchar,
     name varchar,
-    primary key(item_id, name)
+    PRIMARY KEY (item_id, name)
 );
 
-create table Graphs (
+create table graph (
     item_id varchar,
     name varchar,
-    primary key(item_id, name)
+    PRIMARY KEY (item_id, name)
```

Comment:
Could we have more than a `name` ? Weird to have it as part of the `PRIMARY KEY`